barretenberg/cpp/src/barretenberg/sumcheck/instance/prover_instance.hpp

```diff
@@ -92,7 +92,6 @@ template <class Flavor> class ProverInstance_ {
 
         sorted_polynomials = construct_sorted_list_polynomials<Flavor>(circuit, dyadic_circuit_size);
 
-        verification_key = std::make_shared<VerificationKey>(proving_key);
```

Comment:
If the verification key was removed from the prover, should we keep the field?

Reply:
that's correct, we don't need to, removed